src/weatherControl.js

```diff
@@ -8,6 +14,34 @@ const populateWeather = (results) => {
   desc.innerHTML = results.weather[0].description;
   img.src = `http://openweathermap.org/img/w/${results.weather[0].icon}.png`;
 };
+
+const backgroundChanger = (results) => {
+  switch (results.weather[0].main) {
+    case 'Clear':
+      document.body.style.backgroundImage = `url(${clear})`;
+      break;
+    case 'Clouds':
+      document.body.style.backgroundImage = `url(${cloudy})`;
+      break;
+    case 'Rain':
+      document.body.style.backgroundImage = `url(${rain})`;
+      break;
+    case 'Drizzle':
+      document.body.style.backgroundImage = `url(${rain})`;
+      break;
+    case 'Mist':
+      document.body.style.backgroundImage = `url(${rain})`;
+      break;
+    case 'Thunderstorm':
+      document.body.style.backgroundImage = `url(${storm})`;
+      break;
+    case 'Snow':
+      document.body.style.backgroundImage = `url(${snow})`;
+      break;
+    default:
+      break;
+  }
+};
```

Comment:
## Application Mandatory Changes
- [ ] As pointed out earlier, the application needs to have a `toggle` control to determine if the result will be shown in Celsius or Fahrenheit *(There are two valid solutions: you might choose to make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit)* ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [ ] Whichever route you will choose, modularisation of the codebase will be required and thus it will be also expected of you to try and keep the application logic separated from DOM manipulation tasks in order to achieve the `separation of concerns` this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright offers a hint to this. 

happy hacking 💻